backend/src/routes.js

```diff
@@ -1,26 +1,31 @@
 const express = require('express');
-const {User} = require("./Schema");
+const { User } = require("./Schema");
 const router = express.Router();
 
-
-router.get('/users', (req, res) => {
-    res.send(`<h1>No users yet :(</h1>`)
+//Route to get all users
+router.get('/users', async (req, res) => {
+    const users = await User.find({});
+    try {
+        res.status(200).send(users)
```

Comment:
Just make sure to wrap both responses inside JSON objects respectively as noted in the acceptance criteria and I will give you some :beer:

Reply:
Good morning team
Just a reminder to "hide" sensitive data, like password for example
Cheers